encoding/preparation.md

```diff
@@ -286,3 +286,5 @@ Just extract it somewhere.
 ---
 
 [^1]: It should be noted that the author strongly disagrees with this sentiment. The two have a lot in common, and any capable Avisynth encoder could reach a similar leven in Vapoursynth within a few months, maybe even weeks. At least I'm honest, okay?
+
+[^2]: https://github.com/Irrational-Encoding-Wizardry/guide.encode.moe/issues/1
```

Comment:
Instead of adding a link to this, we should just fix the text.

Reply:
Agreed, but not really sure what the author of https://github.com/jpsdr/x264/releases meant. I didn't want to put something wrong there because I'm not sure exactly what happened on the release.

Reply:
Well, I guess something is better than nothing.